config/language_releases.yaml

```diff
@@ -0,0 +1,22 @@
+english:
+  - version: "0.0.1"
+    modules:
+      - module: spacy
+        name: en_core_web_md
+        command:
+          - -m
+          - spacy
+          - download
+          - en_core_web_sm
+      - module: basic
+        name: en_rpa_simple
+        tar: 'https://github.com/rpa-tomorrow/model-releases/releases/download/en_rpa_simple-0.0.1/en_rpa_simple-0.0.1.tar.gz'
+      - module: reminder
+        name: en_rpa_simple_reminder
+        tar: 'https://github.com/rpa-tomorrow/model-releases/releases/download/en_rpa_simple_reminder-0.0.1/en_rpa_simple_reminder-0.0.1.tar.gz'
+      - module: schedule
+        name: en_rpa_simple_calendar
+        tar: 'https://github.com/rpa-tomorrow/model-releases/releases/download/en_rpa_simple_calendar-0.0.1/en_rpa_simple_calendar-0.0.1.tar.gz'
```

Comment:
Should probably update to calendar 0.0.3 here. 

https://github.com/rpa-tomorrow/model-releases/releases/download/en_rpa_simple_calendar-0.0.3/en_rpa_simple_calendar-0.0.3.tar.gz